src/lambdas/sendEmailRequestToNotify/sendEmailToNotify.ts

```diff
@@ -24,8 +27,12 @@ export const sendEmailToNotify = async (
 
   const responseInfo = {
     status: response.status,
-    emailSentTo: requestDetails.email,
+    emailSentTo: JSON.parse(response.config.data).email_address,
```

Comment:
QQ: does the response or responseInfo give us the downloadhash or the download url? This would be useful in the integration test.

Reply:
No. This isn't what it's for. This is mimicking the response that Notify gives to our lambda that sends an email, something that the tests will not see (because it's within the production code). The URL you're talking about is in the *request* to this mocked endpoint, not the response.

The next thing I'm going to do is store the request body of the request that was made to this mock endpoint, so that we have a way to get it back later in a test, via an HTTP endpoint that I'll also build.